python/cuml/solvers/qn.pyx

```diff
@@ -418,9 +418,42 @@ class QN(Base):
 
         return cudf.Series(preds)
 
+    def score(self, X, y):
+        from cuml.metrics import accuracy_score
```

Comment:
wouldn't this import be better outside of the method/class alongside the regular imports?